earth_enterprise/src/common/khxml/khxml.cpp

```diff
@@ -284,7 +377,7 @@ bool
 WriteDocumentToString(DOMDocument *doc, std::string &buf) throw()
 {
   bool success = false;
-  InitializeXMLLibrary();
+  //InitializeXMLLibrary();
```

Comment:
I'm not familiar at all with how and when xml is used but it looks possible for this to be called before any other functions which initialize the library.  I'm also assuming `getDOMImplementation` is a Xerces function.  So could this cause a call to an uninitialized library?

Reply:
A DOMDocument is passed to this function and you can't get one of those without initializing Xerces, so this is safe.